pkg/cmd/run.go

```diff
@@ -65,6 +66,7 @@ doppler run --command "YOUR_COMMAND && YOUR_OTHER_COMMAND"`,
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		enableFallback := !utils.GetBoolFlag(cmd, "no-fallback")
+		enableCache := enableFallback && !utils.GetBoolFlag(cmd, "no-cache")
```

Comment:
Just a thought, but now this variable sounds like this not enabled by default, where I think it is by default enabled right?
Should we name the variables so they make sense with the default?

Reply:
I generally prefer to use variables that indicate whether something is enabled, rather than the reverse, regardless of the default value. We do the same with `enableFallback` (which is enabled by default). This is just personal preference though.